wp-staging-hooks.php

```diff
@@ -483,6 +490,7 @@ function wpstg_backup_files_ignore_files_w_extension_bigger_than($default){
   
     /**
      * Backup: Exclude folders from being included in a backup
+     * E.g. This examples excludes the folder wp-content/cache
```

Comment:
May be use this example: `WP_CONTENT_DIR . '/uploads/wp-staging/cache`, I think we need to highlight the fact that this filter works with absolute path not relative one!

Reply:
Done. Afaik: Folder uploads/wp-staging/ including its cache folder is never included in backup and always excluded. So I've changed the example.

Reply:
> Done. Afaik: Folder uploads/wp-staging/ including its cache folder is never included in backup and always excluded. So I've changed the example.

yes, you are totally right! I meant to change the exemple while I was copying it... thanks!